src/utils/entities.js

```
@@ -0,0 +1,39 @@
+ *
+ * This function is used to prepare the record for display,
+ * so failure to deserialise the JSON blog is ignored and it's
+ * the caller's responsibility to check and handle that manually.
+ *
+ * @param {Object} entity entity record
+ * @returns {Object}
+ */
+export const prepareEntityForTable = (entity) => {
+  const entries = Object.entries(entity).map(([k, v]) => [k.replace('_', '-'), v])
+  const ie = Object.fromEntries(entries)
+  delete ie.json
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)